cdb2api/cdb2api.c

```diff
@@ -908,13 +908,13 @@ static void read_comdb2db_cfg(cdb2_hndl_tp *hndl, FILE *fp, char *comdb2db_name,
                 tok = strtok_r(NULL, " :,", &last);
                 if (tok)
                     cdb2_cache_ssl_sess = !!atoi(tok);
-            } else if (strcasecmp("allow_pmux_route", tok) == 0) {
+            } else if (strcasecmp("cdb2_allow_pmux_route", tok) == 0) {
```

Comment:
Please do not change the config name.

Reply:
This was unintentional, thanks for catching.